### PR TITLE
{bio}[foss/2022a] CellBender v0.2.2

### PR DESCRIPTION
**Pull request by lexming**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/7aee81e32580dc43e4c85220ff981afd for a full test report.

File: easybuild/easyconfigs/c/CellBender/CellBender-0.2.2-foss-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    ('scikit-learn', '1.1.2'),
+    ('matplotlib', '3.5.2'),
+    ('anndata', '0.8.0'),
+    ('PyTables', '3.8.0'),
+    ('pyro-ppl', '1.8.4'),
+]
+
+exts_list = [
+    ('cellbender', version, {
+        'preinstallopts': "sed -i '/^sphinx/d' REQUIREMENTS.txt && ",
```
Comment (verdurin) [inline easybuild/easyconfigs/c/CellBender/CellBender-0.2.2-foss-2022a.eb:26]: @lexming could you add a comment about this please?

**Review by verdurin — DISMISSED** [review]

File: easybuild/easyconfigs/c/CellBender/CellBender-0.2.2-foss-2022a.eb (hunk 1 above)
Comment (lexming) [reply easybuild/easyconfigs/c/CellBender/CellBender-0.2.2-foss-2022a.eb:27]: sure, done in https://github.com/easybuilders/easybuild-easyconfigs/pull/19296/commits/01e492f27e41fe8a03fa1d9972d869a68db111d3

**Comment by lexming** [comment]
@boegelbot please test @ jsc-zen2

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**FAILED**
Build succeeded for 0 out of 3 (1 easyconfigs in total)
node040.viking2.yor.alces.network - Linux Rocky Linux 8.8, x86_64, AMD EPYC 7643 48-Core Processor, Python 3.6.8
See https://gist.github.com/jfgrimm/9cb397481d39f7c2c66864f36e5ecb74 for a full test report.

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in total)
node040.viking2.yor.alces.network - Linux Rocky Linux 8.8, x86_64, AMD EPYC 7643 48-Core Processor, Python 3.6.8
See https://gist.github.com/jfgrimm/65bac4a9795a23c1f87c8dfe6f05053b for a full test report.

**Comment by jfgrimm** [comment]
Going in, thanks @lexming!

**Pull request merged**